### PR TITLE
Upgrade to pdf.js 1.5.188

### PR DESCRIPTION
**Pull request by jheth**
I wanted to get this up to date with the latest stable pdf.js. 

This is loading and working for me but I am not entirely about the introduction of pdf.worker.js. I set the workerSrc which was previously commented out.

@senny  Do you see issues with upgrading?

### CODE REVIEW

File: public/pdfjs/web/viewer.html [hunk 1]
```diff
@@ -14,25 +14,29 @@
...
+    Copyright 1990-2015 Adobe Systems Incorporated.
+
+See https://github.com/adobe-type-tools/cmap-resources
 -->
 <html dir="ltr" mozdisallowselectionprint moznomarginboxes>
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
     <meta name="google" content="notranslate">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```
Comment (jheth) [inline public/pdfjs/web/viewer.html]: Do these changes need to be applied to `_viewer.html.erb`?

**Comment by senny** [comment]
`pdf.worker.js` is not particularly new. I've always used the combined js file.

**Comment by senny** [comment]
There have been code edits in the existing files. If you update you must make sure that these edits are persisted. /cc @siegy22

**Comment by jheth** [comment]
@senny Is this something you guys want to do? Is there more I can do?

**Comment by senny** [comment]
@jheth yes, generally we used to upgrade the PDF.js versions here and then. The difficult part is that the distributed files have been modified in this project. The only way to find out what modifications that are is by looking at the git history. I do vaguely remember something about rendering links.

Given the patches made in this project are applied to the newest version it's ok to upgrade PDF.js.

Unfortunately I am no longer working on projects that make use of this gem. @siegy22 should still be on that and will hopefully get back to you.

**Comment by siegy22** [comment]
I am very sorry for the extrem long delay, I hadn't this much time and was on vacation the last 2 weeks. Going to look over this tomorrow or this weekend!

**Comment by OlivierGrimard** [comment]
+ 1

**Comment by siegy22** [comment]
See #34

**Pull request closed without merging**